deploy-manage/monitor/logging-configuration/enabling-kibana-audit-logs.md

```diff
@@ -1,6 +1,11 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/kibana/current/xpack-security-audit-logging.html
+applies:
+  hosted: all
+  ece: all
+  eck: all
```

Comment:
same with this

Reply:
also similar response. whatever we agreed in the previous topic we will apply it here :)